tests/Integration/PageSaveIntegrationTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\PersistentPageIdentifiers\Tests\Integration;
+
+use ProfessionalWiki\PersistentPageIdentifiers\Adapters\DatabasePersistentPageIdentifiersRepo;
+use ProfessionalWiki\PersistentPageIdentifiers\Application\PersistentPageIdentifiersRepo;
+use ProfessionalWiki\PersistentPageIdentifiers\Tests\PersistentPageIdentifiersIntegrationTest;
+
+/**
+ * @group Database
+ */
+class PageSaveIntegrationTest extends PersistentPageIdentifiersIntegrationTest {
```

Comment:
I created a more generic integration test here, but basically they touch `PersistentPageIdentifiersHooks::onPageSaveComplete()`.

It seemed a bit cleaner to just enable/disable the hook and testing at a higher level, as opposed to disabling the hook and calling `PersistentPageIdentifiersHooks::onPageSaveComplete()` manually.